submissions/ayodeleayobami/landingpage/task.html

```diff
@@ -0,0 +1,113 @@
+	   <title>Hompage</title>
+	   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+	   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	   <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+     <link rel="stylesheet" href="style.css">	
+</head>
+<body>
+<div class="container">
+  <div class="main">	
+     <nav>
+      <div class="logo">
+        <img src="images/logo.jpg">	
+     </div>
+   <ul>
+        <li><a href="#">HOME</a></li>
+        <li><a href="#">ABOUT US</a></li>
+        <li><a href="#">COLLECTION</a></li>
+        <li><a href="#">SHOP</a></li>
+        <li><a href="#">CONTACT</a></li>
+    </ul>
+  </nav>
+</div>
+<!-------slider------->
+<div id="slider">
+	  <div id="headerSlider" class="carousel slide" data-ride="carousel">
+      <ol class="carousel-indicators">
+         <li data-target="#headerSlider" data-slide-to="0" class="active"></li>
+         <li data-target="#headerSlider" data-slide-to="1"></li>
+         <li data-target="#headerSlider" data-slide-to="2"></li>
+  </ol>
+  <div class="carousel-inner">
+    <div class="carousel-item active">
+      <img class="d-block w-100" src="images/shoe1.jpg">
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="images/shoe2.jpg">
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="images/shoe3.jpg">
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="images/shoe4.jpg">
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="images/shoe5.jpg">
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="images/shoe6.jpg">
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="images/shoe7.jpg">
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="images/shoe8.jpg">
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="images/shoe9.jpg">
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="images/shoe10.jpg">
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="images/shoe11.jpg">
+    </div>
+  <a class="carousel-control-prev" href="#headerSlider" role="button" data-slide="prev">
+    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+    <span class="sr-only">Previous</span>
+  </a>
+  <a class="carousel-control-next" href="#headerSlider" role="button" data-slide="next">
+    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+    <span class="sr-only">Next</span>
+  </a>
+  </a>
+</div>
```

Comment:
The indentations here are not quite clear. Indent an inner item with at least 4 spaces as shown below
```
<div>
      <div>inner div</div>
</div>

Reply:
I made corrections.

Reply:
@joymeetme do you use any special editor for indentation or do you do it manually? your files are still not properly indented. what IDE/editor do you use?